sys/src/9/amd64/l64v.S

```diff
@@ -253,10 +253,7 @@ wrmsr:
 
 .global invlpg
 invlpg:
-#	MOVQ	%rdi, va+0(FP)
-
-#	INVLPG	va+0(FP)
-
+	invlpg 0(%rdi)
```

Comment:
Drop the `0`? It shouldn't be necessary.

Reply:
good point. done.